charts/access-request/values.yaml

```
@@ -79,3 +79,4 @@ parameters:
     workers: 1
 
 shareProcessNamespace: false
+imagePullSecretName: my-registry-secret
```

```suggestion
imagePullSecretName: []
```
Since this is the default value we should leave it empty.